site/guides/integration-testing.adoc

```diff
@@ -0,0 +1,970 @@
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>fmp</id>
+                <goals>
+                  <goal>resource</goal>
+                  <goal>build</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+----
+
+We do however need to configure FMP explicitly in the `tests` module since that module needs to aggregate the OpenShift resources produced by the
+`resource` goal of FMP of the microservices modules.
+
+Therefore in the `pom.xml` file of the `tests` module we add the following FMP configuration:
+
+[source,xml]
+----
+<profiles>
+    <profile>
+        <id>openshift-it</id> #<1>
+        <build>
+          <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>fabric8-maven-plugin</artifactId>
+                <executions>
+                  <execution>
+                    <goals>
+                      <goal>resource</goal>
+                    </goals>
+                  </execution>
+                </executions>
+                <configuration>
+                  <profile>aggregate</profile> #<2>
+                </configuration>
+                <dependencies>
+                  <dependency> #<3>
+                    <groupId>io.openshift.booster</groupId>
+                    <artifactId>spring-boot-cache-greeting</artifactId>
+                    <version>${project.version}</version>
+                  </dependency>
+                  <dependency>
+                    <groupId>io.openshift.booster</groupId>
+                    <artifactId>spring-boot-cache-cutename</artifactId>
+                    <version>${project.version}</version>
+                  </dependency>
+                </dependencies>
+            </plugin>
+          </plugins>
+        </build>
+    </profile>
+</profiles>
+----
+
+<1> The `tests` module only has integration test code, so we introduce all the FMP configuration to the `openshift-it` profile instead of the `openshift` profile.
+Because of the way we execute the maven command (`mvn clean verify -Popenshift,openshift-it`) the configuration in the `openshift-it` profile of the module
+overrides the configuration in the `openshift` profile of the parent pom.
+
+<2> Using FMP's link:https://maven.fabric8.io/#profiles-predefined[aggregate] profile makes it search for OpenShift resource files inside
+the jars that are available via the dependencies of the module or the dependencies of the plugin itself. Any such resources files will be
+aggregated into a single file and placed in the standard `target/classes/META-INF/fabric/openshift.yml` location we have seen in previous sections. +
+It's important to note that FMP will not attempt to generate any other OpenShift resources when this profile is enabled.
```

Comment:
Maybe use: 
IMPORTANT: asciidoc callout?

Reply:
Didn't know that one, thanks!!!